tests/Dekaf.Tests.Unit/Protos/confluent/meta.proto

```
@@ -0,0 +1,41 @@
+  repeated Rule rules = 4;
+}
+
+message Rule {
+  string name = 1;
+  string doc = 2;
+  string expr = 3;
+  string sql = 4;
+}
+
+extend google.protobuf.FileOptions {
+  Meta file_meta = 1088;
```

Buf (1.72.0)

[error] field number `1088` used more than once

(COMPILE)